src/services/iiif-parser.js

```diff
@@ -299,3 +289,42 @@ export function getPoster(manifest) {
   let posterUrl = parseManifest(manifest).getThumbnail()['id'];
   return posterUrl;
 }
+
+/**
+ * Parse 'start' property in manifest if it is given
+ * In the spec there are 2 ways to specify 'start' property:
+ * https://iiif.io/api/presentation/3.0/#start
+ * Cookbook recipe for reference: https://iiif.io/api/cookbook/recipe/0015-start/
+ * @param {Object} manifest
+ * @returns {Object}
+ */
+export function getCustomStart(manifest) {
+  if (!parseManifest(manifest).getProperty('start')) {
+    return null;
+  }
+  let currentCanvasIndex = null;
+  let startProp = parseManifest(manifest).getProperty('start');
+
+  let getCanvasIndex = (canvasId) => {
+    const canvases = canvasesInManifest(manifest);
+    const currentCanvasIndex = canvases
+      .map(function (c) {
+        return c.canvasId;
+      })
+      .indexOf(canvasId);
+    return currentCanvasIndex;
+  };
+  if (startProp) {
+    switch (startProp.type) {
+      case 'Canvas':
+        currentCanvasIndex = getCanvasIndex(startProp.id);
+        return { type: 'C', canvas: currentCanvasIndex, time: 0 };
+      case 'SpecificResource':
+        currentCanvasIndex = getCanvasIndex(startProp.source);
+        let customStart = startProp.selector.t;
+        return { type: 'SR', canvas: currentCanvasIndex, time: customStart };
+    }
+  } else {
+    return null;
+  }
```

Comment:
This `else` should be unreachable because of the early return at the top of the method so can it be removed?

Reply:
Absolutely! :+1: